packages/datetime/src/dateInput.tsx

```diff
@@ -73,9 +73,10 @@ export interface IDateInputProps extends IDatePickerBaseProps, IDateFormatProps,
 
     /**
      * Called when the user selects a new valid date through the `DatePicker` or by typing
-     * in the input.
+     * in the input. The second argument is true if and only if the user clicked on a date in the
+     * calendar; it will be false if the date was changed by choosing a new month or year.
```

Comment:
@qcz i just edited these docs a bit. question: is the arg true if the user types in a new date in the input?

Reply:
+1 to `isUserChange`, please do that!

Reply:
@giladgray: done!